src/toggle-button/internal.tsx

```diff
@@ -27,7 +27,7 @@ export const InternalToggleButton = React.forwardRef(
       onChange,
       className,
       ...rest
-    }: ToggleButtonProps,
+    }: ToggleButtonProps & { __title?: string },
```

Comment:
This is needed to hide the HTML title of the button when it is used in the button group. That is because the title is not needed when the tooltip is present.